internal/pkg/kernel/kernel.go

```diff
@@ -20,10 +20,10 @@ func NewDefaultCmdline() *cmdline {
 	cmdline.Append("pti", "on")
 	// TODO(andrewrynhard): Add slub_debug=P. See https://github.com/talos-systems/talos/pull/157.
 	cmdline.Append("consoleblank", "0")
-	cmdline.Append("console", "ttyS1,115200n8")
+	//cmdline.Append("console", "ttyS1,115200n8")
```

Comment:
Leaving this comment in for discussion. I'm not sure of the best place to migrate these parameters to.

Reply:
These are currently called inline:

```
internal//app/init/internal/platform/iso/iso.go:	cmdline := kernel.NewDefaultCmdline()
internal//app/init/internal/platform/baremetal/baremetal.go:	cmdline := kernel.NewDefaultCmdline()
internal//app/init/internal/platform/cloud/packet/packet.go:	cmdline := kernel.NewDefaultCmdline()
```

Reply:
I wonder if we need `--append-kernel-arg` and `--set-kernel-arg`. Then use `cmdline.Set(...)`. 🤔 not the prettiest. Or somehow in the `NewDefaultCmdline` we handle this.

Reply:
The quick win is that we just be explicit in our image creation (i.e. just use `-e` in the `Makefile`).